### DISCUSSION TITLE
Add label from the API

### DISCUSSION THREAD

**Discussion by sr258** [WeblateOrg/weblate#9840] [Q&A]
Is it possible to add a label to a string with the API? The [documentation on the PATCH /api/units/](https://docs.weblate.org/en/latest/api.html#patch--api-units-(int-id)-) endpoints doesn't list labels as a valid part of the request object.

Is the JSON object in the request really limited to state, target, explanation and extra_flags?

**Comment by nijel** [comment]
Labels are currently read-only in the API.
